packages/manager/src/features/Linodes/LinodesDetail/LinodeSettings/LinodeSettingsMaintenancePolicyPanel.tsx

```diff
@@ -89,7 +98,7 @@ export const LinodeSettingsMaintenancePolicyPanel = (props: Props) => {
             name="maintenance_policy"
             render={({ field, fieldState }) => (
               <MaintenancePolicySelect
-                disabled={isReadOnly}
+                disabled={isReadOnly || !regionSupportsMaintenancePolicy}
```

Comment:
Should we add a tooltip explaining why the component is disabled?


```suggestion
                disabled={isReadOnly || !regionSupportsMaintenancePolicy}
                textFieldProps={{
                  tooltipText: isReadOnly
                    ? getRestrictedResourceText({
                        resourceType: 'Linodes',
                        isSingular: true,
                        action: 'edit',
                        includeContactInfo: false,
                      })
                    : !regionSupportsMaintenancePolicy
                      ? "This Linode's region does not support maintenance policies."
                      : undefined,
                }}
```

Related to the bug, we should probably remove `You don't have permission to change this setting` copy  from the `MaintenancePolicySelect` component and have the parent provide it **or** maybe we add a new prop to `MaintenancePolicySelect` called `hasPermission` or something

Reply:
Good call thanks